widechar_width.h

```diff
@@ -1,3 +1,7 @@
+// SPDX-FileCopyrightText: © 2018 ridiculousfish and widecharwidth contributors
+//
+// SPDX-License-Identifier: CC0-1.0
+
```

Comment:
These files (widechar_width.{h,js,py,rs}) are *generated* from the corresponding files in templates/. They include the hash for the template that they were generated from, so instead of adding the header here you need to add it to the template and then run generate.py again.